_docs/setup/kubernetes/multicluster-install.md

```diff
@@ -0,0 +1,161 @@
+kind: Cluster
+metadata:
+  name: "falkor07"
+  annotations:
+    config.istio.io/pilotEndpoint: "192.168.1.1:9080"
+    config.istio.io/platform: "Kubernetes"
+    config.istio.io/pilotCfgStore: True
+    config.istio.io/accessConfigFile: "falkor07.kube.conf"
+  spec:
+    kubernetesApiEndpoints:
+      serverEndpoints:
+        - clientCIDR: "10.23.230.0/28"
+          serverAddress: "10.23.230.11"
+```
+
+A unique configmap is required for each cluster.  In the above example,
+`falkor07.kube.conf` is the Istio control plane credential file. The
+config.istio.io/pilotCfgStore = `True` since this is the Istio control plane.
+For remotes, pilotCfgStore should = `False`.
+
+**Important**: The impelementation only uses the
+`config.istio.io/pilotCfgStore` and `config.istio.io/accessConfigFile`
+annotations, although every other annotation and spec is validated for
+correct syntax.  They may be set to dummy values, as long as they can
+be validated correctly.
+
+If the prequisites are met, the credentials for each Kubernetes cluster
+will also be present in `$HOME/multicluster`.
+
+Assemble and create the configmap from these files in the Kubernetes cluster
+where the Istio control plane is operational:
+
+```bash
+kubectl create configmap -n istio-system multicluster -f $HOME/multicluster
+```
+
+## Deploy on the remote cluster
+
+Multicluster makes **one** Istio service mesh from all Kubernetes clusters
+that participate in multicluster.  Multicluster uses **one** Kubernetes
+cluster to run the Istio control plane.  **Any reasonable** number of remote
+Kubernetes clusters may be attached to the **one** Istio control plane.
+
+### Use kubectl with Helm to connect the remote cluster
+
+1. Use the helm template command on a remote to specify the Istio control plane:
+
+   ```bash
+   helm template install/kubernetes/helm/istio-remote --name istio-remote --set pilotEndpoint=`pod_ip_of_pilot_master` --set mixerEndpoint=`pod_ip_of_mixer_master` > $HOME/istio-remote.yaml
+    ```
+
+1. Instantiate the remote cluster's connection to the Istio control plane:
+
+   ```bash
+   kubectl create -f $HOME/istio-remote.yaml
+   ```
+
+### Use Helm and Tiller to connect the remote cluster
+
+1. If a service account has not already been installed for Helm, please
+install one:
+
+   ```bash
+   kubectl create -f install/kubernetes/helm/helm-service-account.yaml
+   ```
+
+1. Initialize Helm:
+
+   ```bash
+   helm init --service-account tiller
+   ```
+
+1. Install the Helm chart:
+
+   ```bash
+   helm install install/kubernetes/helm/istio-remote --name istio-remote --set pilotEndpoint=pod_ip_of_pilot_master --set mixerEndpoint=pod_ip_of_mixer_master`
+   ```
+
+### Mandatory Helm configuration parameters
+
+The isito-remote Helm chart requires the configuration of two specific variables defined in the following table:
+
+**Note** The `pilotEndpoint` and `mixerEndpoint` can also use DNS resolution, assuming DNS is setup to resolve
```

Comment:
I am not sure what you are trying to say here.   The remote cluster will use kube-DNS to try to resolve the remote pilot (istio-pilot) and mixer (istio-mixer) endpoints.  It doesn't work any other way with the current code.   So each cluster must have a kube-dns entry for that service name.  how the cluster running pilot and mixer map those to an IP address is the part that has options.

Reply:
see review comment: https://github.com/istio/istio/pull/4655#discussion_r178597476 for rationale on this Note.  The **Note** can be removed, since nothing actually validates it works and has not been validated by a human.

Reply:
sounds like a good approach.  Thanks for the suggestion.

Reply:
clairified.